concordium-node/src/configuration.rs

```diff
@@ -225,6 +225,13 @@ pub struct BakerConfig {
         long = "import-blocks-from",
         help = "Path to a file exported by the database exporter"
     )]
+    #[structopt(
+        long = "max-expiry-duration",
+        help = "Maximum allowed time difference between now and a transaction's expiry time in \
+                milliseconds",
```

Comment:
Nobody will have a max expiry granularity in milliseconds so this should be probably changed.